### ISSUE TITLE
Asset compilation - Coffee, Sass, Images

### ISSUE THREAD

**Issue by drujensen** [johnjansen/crystal-libraries-needed#7]
Yarn and web brick

**Comment by elorest** [comment]
Yeah. I think that all of this exists. We just need to glue it together.

**Comment by TheLonelyGhost** [comment]
SASS support might come from C-bindings into libsass, which might have [slightly different](http://sassbreak.com/ruby-sass-libsass-differences/) compatibility than ruby sass.

CoffeeScript might require just deferring to node or npm package for it.

Image transformation is just one of a few libraries: imagemagick, GraphicsMagick, optipng, etc.

I assume the idea is to have an alternative to the rails asset pipeline, at least in part?

**Comment by RX14** [comment]
Webpack already exists as a feature rich solution with a big ecosystem. Instead of attempting to reinvent the wheel, why not look at ways to integrate webpack as seamlessly as possible in crystal projects.
